docs/src/8-apis/README.md

```diff
@@ -17,6 +17,32 @@ Through these various API routes, you can access and integrate PrestaShop store
 Our team is here to help you get started with the implementation of PrestaShop CloudSync APIs. Please [click here](https://meetings.hubspot.com/esteban-martin3/prestashop-new-framework-integration-meeting) to set up a meeting with us before proceeding with the integration of your module.
 :::
 
+## Getting an Access Token
+
+To query the PrestaShop Integration Framework APIs, you first need to receive a JWT from the Authorization Server.
+
+1. Get in touch with PrestaShop Solution Engineers (partners-soleng@prestashop.com) to get your client credentials.
+
+2. Call the **Account Authorization Server** to get your JWT.
+
+    **Authorization Server URL (production) :** [https://oauth.prestashop.com/](https://oauth.prestashop.com/) 
+
+    Below is an example of a request to the Authorization Server:
```

Comment:
As OAuth2 is standard, we should let the developers use a library instead of developing the standard again.

```suggestion
    You can use the library you want for your favorite language. Here is a list of available code libraries at this [link](https://oauth.net/code/). If you want to make this with a curl, below is an example of a request to the Authorization Server:
```